library/liquibase

```diff
@@ -0,0 +1,7 @@
+Maintainers: Jake Newton <docker@liquibase.com> (@jnewton03)
+GitRepo: https://github.com/liquibase/docker.git
+
+Tags: 4.25, 4.25.1, 4.25-alpine, 4.25.1-alpine, latest
+GitFetch: refs/heads/main
+GitCommit: 5e9438c2fdfabbce058946229e3df2bce958aa32
+Directory: .
```

Comment:
Presumably the `-alpine` tags would have a different branch, commit, and/or directory.

Reply:
@whalelines We addressed the latest items and we also added a new entry for `alpine` tags.